pyslackersweb/contexts.py

```diff
@@ -29,4 +34,5 @@
 async def client_session(app: web.Application) -> None:
     async with ClientSession(raise_for_status=True) as session:
         app["client_session"] = session
+        app["slack_client"] = SlackAPI(token=app["slack_token"], session=session)
```

Comment:
This should be an independent context (or setup signal). It depends on this one, but shouldn't expand what the `client_session` cleanup_ctx does.

Reply:
I was afraid you where going to say that.

Reply:
I wasn't sure on the ordering of processing, but it looks like cleanup_ctx always is applied first on setup and cleanup:

https://github.com/aio-libs/aiohttp/blob/c8cbbcc3b979e13305c187929f4708c528c19084/aiohttp/web_app.py#L117-L119